### PR TITLE
javascript/resistor-color-duo: add minimal mentor notes

### PR DESCRIPTION
**Pull request by SleeplessByte**
Add minimal mentor notes for `resistor-colors`.

### CODE REVIEW

File: tracks/javascript/exercises/resistor-colors/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+}
+```
+
+A student may choose to enter the `function` into `map`:
+
+```javascript
+colors.map(COLORS.indexOf.bind(COLORS))
+```
+
+A student may use also use an arrow function expression instead of a function declaration, as well as using an
```
Comment (tejasbubane) [inline tracks/javascript/exercises/resistor-colors/mentoring.md]: This still needs fixing - `may also use an arrow function`

Comment (SleeplessByte) [reply tracks/javascript/exercises/resistor-colors/mentoring.md]: I don't know what happened there

File: tracks/javascript/exercises/resistor-colors/mentoring.md [hunk 2]
```diff
@@ -12,15 +12,15 @@ export const COLORS = [
 export function value(colors) {
   return Number(colors.map(color => COLORS.indexOf(color)).join(''))
 }
-```
+```test
```
Comment (tejasbubane) [inline tracks/javascript/exercises/resistor-colors/mentoring.md]: This is not required.

File: tracks/javascript/exercises/resistor-colors/mentoring.md [hunk 3]
```diff
@@ -78,7 +78,10 @@ Approve the solution, but challenge them to solve it using `map`, as listed belo
 
 ### Common suggestions
 
-Here are a few common suggestions based on student's solutions. You may always suggest they re-use code from their
+Here are a few common suggestions based on student's solutions. You may always suggest they re-
+
+
+code from their
```
Comment (tejasbubane) [inline tracks/javascript/exercises/resistor-colors/mentoring.md]: This needs fixing - `re-use code from their solution`

Comment (SleeplessByte) [reply tracks/javascript/exercises/resistor-colors/mentoring.md]: I'm weeping because I don't even know HOW this got like this.

**Comment by tejasbubane** [comment]
Hey @SleeplessByte can you rename this to `resistor-color-duo`?

**Pull request merged**